test/PHPUnit/Controller/MemoryLeakTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-test for the canonical source repository
+ * @copyright Copyright (c) 2005-2018 Zend Technologies USA Inc. (https://www.zend.com)
+ * @license   https://github.com/zendframework/zend-test/blob/master/LICENSE.md New BSD License
+ */
+namespace ZendTest\Test\PHPUnit\Controller;
+
+use Zend\Test\PHPUnit\Controller\AbstractControllerTestCase;
+
+class MemoryLeakTest extends AbstractControllerTestCase
+{
+    public static $mem_start;
+
+    public static function setUpBeforeClass()
+    {
+        self::$mem_start = memory_get_usage(true);
+    }
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        $this->setApplicationConfig(
+            [
+                'modules'                 => [
+                    'Zend\\Router',
+                ],
+                'module_listener_options' => [],
+            ]
+        );
+    }
```

Comment:
@ezimuel I think now we can remove setUp method completely as we setApplicationConfiguration in the test, right?

Reply:
@webimpress you right, I removed it.